chatops4s-discord/src/main/scala/api/DiscordInbound.scala

```diff
@@ -1,29 +1,25 @@
 package api
 
-import api.Server.getClass
 import cats.effect.IO
-import com.typesafe.scalalogging.Logger
+import com.typesafe.scalalogging.{Logger, StrictLogging}
 import models.{Button, ButtonInteraction, InboundGateway, InteractionContext}
 
 import scala.collection.concurrent.TrieMap
 
-class DiscordInbound(verbose: Boolean = false) extends InboundGateway {
-  private final val logger = Logger(getClass.getName)
+class DiscordInbound extends InboundGateway, StrictLogging {
   val handlers: TrieMap[String, InteractionContext => IO[Unit]] = TrieMap.empty[String, InteractionContext => IO[Unit]]
 
   override def registerAction(handler: InteractionContext => IO[Unit]): ButtonInteraction = {
-    if (verbose) logger.info("Registered action")
+    logger.whenInfoEnabled {
```

Comment:
this is already baked in the logger logic. All you need to do is to blindly call `logger.info()`